src/components/editor/editor-pane/upload-handler.ts

```diff
@@ -8,12 +8,17 @@ import { Editor } from 'codemirror'
 import i18n from 'i18next'
 import { uploadFile } from '../../../api/media'
 import { store } from '../../../redux'
+import { supportedMimeTypes } from './tool-bar/utils/upload-image-mimetypes'
 
 export const handleUpload = (file: File, editor: Editor): void => {
   if (!file) {
     return
   }
   const mimeType = file.type
+  if (!supportedMimeTypes.includes(mimeType)) {
+    // this mimetype is not supported
```

Comment:
Do we want to show an error in this case?

Reply:
As we don't show errors if the upload fails, yes but how?

Reply:
Hm, maybe we don't show any errors for now but open an issue to implement such a thing later (maybe based on some kind of toast-system)?

Reply:
sounds reasonable to me

Reply:
would you open that issue then?

Reply:
(link https://github.com/hedgedoc/react-client/issues/816 in it pls)